### ISSUE TITLE
[TW-244] task add "Description (Information)" results in "Description ( Information)"

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#287]
_Dirk Deimeke on 2012-02-10T09:06:40Z says:_ 

 There is a little formatting error with brackets in descriptions.

<pre>
$ task add "abc (cde)"
Created task 51.

$ task 51 minimal

ID Project Description
51         abc ( cde)

1 task</pre>

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2012-02-10T09:06:40Z
Modified: 2014-06-08T14:32:37Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2012-02-11T19:06:22Z says:_ 

 Not sure what to do about this one.  It exposes a flaw in the parser.

**Comment by taskwarrior** [comment]
_David Patrick on 2012-02-11T19:17:55Z says:_ 

 Paul Beckingham wrote:
> Not sure what to do about this one.  It exposes a flaw in the parser.

in SOVIET RUSSIA! BUG FIXES YOU!!!

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2012-02-11T19:19:18Z says:_ 

 Nice.  What I meant was "exposes a big flaw in the parser - one that I don't know how to address yet."

**Comment by taskwarrior** [comment]
_David Patrick on 2012-02-11T19:28:08Z says:_ 

 yeah, caught that, just thought I'd add some levity/perspective, turn the problem on it's ear approach.. keep us posted with your thoughts on scope and implications, Doctor, we'll be in the cafeteria..

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2012-02-11T19:33:19Z says:_ 

 David Patrick wrote:
> ... thought I'd add some levity ...

I appreciate the levity - makes it all more fun.  Otherwise it's all just a big sad list of things I broke or failed to do.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2012-03-14T03:55:18Z says:_ 

 Needs a parser fix, but not at this late stage in 2.0.  Deferring to 2.1.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2012-12-01T20:11:06Z says:_ 

 Will be addressed in the 'parser' release after taskd and 2.3.0.
